crates/avalanche-types/src/subnet/rpc/vm/server.rs

```diff
@@ -977,15 +980,41 @@ where
         _req: Request<Empty>,
     ) -> std::result::Result<Response<vm::VerifyHeightIndexResponse>, tonic::Status> {
         log::debug!("verify_height_index called");
-        Err(tonic::Status::unimplemented("verify_height_index"))
+
+        let inner_vm = self.vm.read().await;
+
+        match inner_vm.verify_height_index().await {
+            Ok(_) => return Ok(Response::new(vm::VerifyHeightIndexResponse { err: 0 })),
+            Err(e) => {
```

Comment:
is there every a scenario where we want to return an error?

https://github.com/ava-labs/avalanchego/blob/master/database/rpcdb/errors.go#L25